inclusive-design/idrc
# feat!: update site architecture with new Trivet features

## Description

Resolves #650, resolves #660, resolves #661, resolves #662, resolves #663, resolves #664.

## Steps to test

Build site, verify that it functions as expected with reference to the current release.

## Additional information

https://github.com/fluid-project/trivet/pull/337

## Related issues

Resolves #650, resolves #660, resolves #661, resolves #662, resolves #663, resolves #664.
